genesyscloud/routing_utilization_label/resource_genesyscloud_routing_utilization_label_schema.go

```diff
@@ -0,0 +1,108 @@
+package routing_utilization_label
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/mypurecloud/platform-client-sdk-go/v130/platformclientv2"
+	"strings"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+const resourceName = "genesyscloud_routing_utilization_label"
+
+// SetRegistrar registers all the resources and exporters in the package
+func SetRegistrar(regInstance registrar.Registrar) {
```

Comment:
You need to register the data source here, that's probably why the routing utilization label data source got removed